src/Cart.js

```diff
@@ -0,0 +1,21 @@
+import App from "./App"
+
+function Cart(props) {
+    const cartItemList = props.cartItems
+
+  return (
+    <>
+      {cartItemList.map((cartFruit, index) => (
+        <li key={index}>
+          <img src={`./assets/icons/${cartFruit.id}.svg`}></img>
+          <p>{cartFruit.name}</p>
+          <button onClick={() => props.handleQuantityDecrease(cartFruit)}>-</button>
```

Comment:
ok cool! I was wondering how to get a click event to call the function properly instead of needing to do a lot of in-line coding